cms/celery.py

```diff
@@ -34,6 +34,5 @@ def alternate_env_tasks(self):
         return {
             'openedx.core.djangoapps.content.block_structure.tasks.update_course_in_cache': 'lms',
             'openedx.core.djangoapps.content.block_structure.tasks.update_course_in_cache_v2': 'lms',
-            'openedx.core.djangoapps.grades.tasks.compute_grades_for_course': 'lms',
```

Comment:
So in light of this morning's conversation, do we want to move this task into openedx?